Sources/PrimerSDK/Classes/Data Models/PrimerContent.swift

```diff
@@ -110,8 +110,8 @@ struct VaultPaymentMethodViewContent {
     var mainTitleText: String { return NSLocalizedString("primer-vault-payment-method-saved-payment-methods",
                                                          tableName: nil,
                                                          bundle: Bundle.primerResources,
-                                                         value: "Saved payment methods",
-                                                         comment: "Saved payment methods - Vault Payment Method (Main title text)") }
+                                                         value: "SAVED PAYMENT METHODS",
```

Comment:
[Feedback]
The fact that it is all uppercase is a UI decision, independent from the text itself. Can we keep lowercase here but use a function to put the string to uppercase when needed?
Same thing would go for `OTHER WAYS TO PAY`

Reply:
Fixed